setup.cfg

```
@@ -1,7 +1,8 @@
 [tool:pytest]
-norecursedirs = .git venv build dist *egg
-addopts = -rfEsxwX --cov readchar
+testpaths = tests
+addopts = -rfEsxwX -s --cov=readchar
```

```suggestion
addopts = -rfEsxwXs --cov=readchar
```

---

the characters following `-r` are options for this flag. See the help for pytest:

> `-r chars` - show extra test summary info as specified by chars: (f)ailed, (E)rror, (s)kipped, (x)failed, (X)passed, (p)assed, (P)assed with output, (a)ll except passed (p/P), or (A)ll. (w)arnings are enabled by default (see --disable-warnings), 'N' can be used to reset the list. (default: 'fE').

Thats why they need to be kept seperate.

Maybe this qould make it more clear?:

```suggestion
addopts = -r fEsxwX -s --cov=readchar
```